about.html

```diff
@@ -0,0 +1,103 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>About</title>
+  <link rel="stylesheet" href="about.css">
+  <link rel="stylesheet" href="//fonts.googleapis.com/earlyaccess/nanumgothic.css">
+
+</head>
+<body>
+  <main class="maincontainer">
+    <header class="header">
+      <ul class="header1">
+        <li><img src="./images/facebook.png" alt="Facebook"></li>
+        <li><img src="./images/twitter.png" alt="Twitter"></li>
+        <li>Korean</li>
+        <li>My Page</li>
+        <li>Logout</li>
+      </ul>
+      <div class="header2">
+        <div class="header25">
+          <nav class="dropdown">
+            <button type="button" id="button" class="button">
+              <img src=".images/hamburguer.png" alt="Menu" onclick="dropmenu()">
+            </button>
+          </nav>            
+      
+          <div class="header225">
+            <a href="#" class="logo"><img src="./images/concert_logo.png" alt="Live Concert"></a>
+            <!-- the ul has to have ulheader2 class to show -->
+            <ul id="dropmenu" class="hide"> 
+              <li class="lix"><a href="#" class="ex" onclick="closemenu()">x</a></li>
+              <li class="about"><a href="./index.html" class="mainpage">Main Page</a></li>
+              <li onclick="closemenu()"><a class="optionsacts" href="#mainprog">Past Live Concerts</a></li>
+              <li class="optionsheader2">Join</li>
+              <li class="optionsheader2">Sponsor</li>
+              <li class="optionsheader2">News</li>
+            </ul>
+          </div>
+
+          <div class="header3">
+            <p class="hello">"Hello! Sharing a world concert in Bolivia"</p>
+            <p class="massive">MASSIVE LIVE CONCERT IN</p>
+            <p class="bolivia">BOLIVIA VERSION 2022</p>
+            <p class="extreme">This event has been the result of many efforts from the Bolivian community in USA and UK who worked with local Bolivian companies in order to coordinate all the logistics and operations. Also to convice the artists to believe in the objectives and to participate in this event.</p>
+            <p class="contact">Please contact us per Email for any further questions about to the first Bolivian Live Concert 2022.</p>
```

Comment:
- [ ] Kindly try to center the text in this section on the about page to match the Behance design.

| Your design | Behance design |
| --- | --- |
|![image](https://user-images.githubusercontent.com/24703171/169205962-770d6ad7-1ce8-4f4e-8895-1681e85df482.png) | ![image](https://user-images.githubusercontent.com/24703171/169206039-10fef190-d7df-4c65-8a7a-a8440bf55d92.png) |

Reply:
ok, it is centered now.